bin/needs-a-genny

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+genny_path = ->(slug) {
+  File.join('exercises', slug, '.meta', 'generator')
+}
+
+no_genny_path = ->(slug) { !Dir.exist?(genny_path[slug]) }
+
+puts Dir.children('exercises')
+  .filter(&no_genny_path).sort
```

Comment:
Methods with negatively phrased names can be confusing.

What about `reject`ing rather than `select`/`filter`ing ?

`reject(&generator_exists?)`

Reply:
I was thinking there must be some method that would do the opposite of filter :smile:

Let me make that change before merging.

Reply:
I wanted a `?` in the filename too, but unfortunately it won't let me do that for a lambda, because it is illegal in variable names :frowning_face: 

I can't figure out how to pass a method defined with `def` in though.  `filter`/`reject` won't pass arguments to it.

Reply:
You can pass methods as objects, if you make the method an object, with `method`.

Reply:
I guess I would prefer not having to wrap it like `method(:generator?)` while passing to `reject`.  If I save it in a variable, then it once again will not have `?` on the end when passed.

I guess I think it is pretty clear now.